include/deal.II/distributed/tria_base.h

```diff
@@ -464,6 +464,24 @@ namespace parallel
     using CellStatus =
       typename dealii::Triangulation<dim, spacedim>::CellStatus;
 
+    /**
+     * Return true if the triangulation has hanging nodes.
+     *
+     * In the context of parallel distributed triangulations, every
+     * processor stores only that part of the triangulation it locally owns.
```

Comment:
Pre-existing, but I think proper English grammar needs to spell this as
```suggestion
     * processor stores only that part of the triangulation it owns locally.
```